File: Java Script - Introduction to Programming/9.9. Functions/03. Printing Triangle.js
function printingTriangle(n) {

    function print(start, end) {
        let result = ''
        for (let i = start; i <= end; i++) {
            result += i + " "
        }
        return result
    }

    for (let i = 1; i <= n; i++) {
        console.log(print(1, i))
    }

    for (let i = n - 1; i > 0; i--) {
        console.log(print(1, i))
    }
}


// printingTriangle(3)

File: Pyrhon - Introduction to Programming/9.9. Functions/03. Printing Triangle.py
def printing_triangle(n):

    def printing(start, end):
        result = ''
        for i in range(start, end + 2):
            result += str(i) + " "

        return result


    for i in range(0,n):
        print(printing(1, i))


    for i in range(n-2,-1,-1):
        print(printing(1, i))


printing_triangle(int(input()))

#3
